poise/hello-world/src/lib.rs

```diff
@@ -0,0 +1,42 @@
+use anyhow::Context as _;
+use poise::serenity_prelude as serenity;
```

Comment:
```suggestion
use poise::serenity_prelude::*;
```

Might need a few other code changes

Reply:
I don't think that is a good idea. Even though it's called prelude, it actually imports a lot more from serenity, including `serenity::Result` [[1]](https://github.com/serenity-rs/poise/blob/aba9c8363ead0e185f4189f0062c7662642c1851/src/lib.rs#L395) [[2]](https://github.com/serenity-rs/serenity/blob/4c9a61c227257ae0d65f6ce84c6b62fbccb59b84/src/lib.rs#L124), which breaks `shuttle_service::main`.

Poise gives `use poise::serenity_prelude as serenity;` as an [example](https://github.com/serenity-rs/poise/blob/aba9c8363ead0e185f4189f0062c7662642c1851/src/lib.rs#L358).

Reply:
Gotcha! Thanks for the responses on these